clang/include/clang/Basic/AttrDocs.td

```diff
@@ -2581,31 +2581,32 @@ def SYCLIntelUseStallEnableClustersAttrDocs : Documentation {
   let Category = DocCatFunction;
   let Heading = "intel::use_stall_enable_clusters";
   let Content = [{
-When applied to a lambda or function call operator (of a function object)
-on device, this requests, to the extent possible, that statically-scheduled
-clusters handle stalls using a stall-enable signal to freeze computation
-within the cluster. This attribute is ignored on the host.
+When applied to a lambda function, function definition, or function call
+operator (of a function object) on device or host, this requests, to the
+extent possible, that statically-scheduled clusters handle stalls using a
+stall-enable signal to freeze computation within the cluster. The attribute
+is not ignored on host mode but only generates metadata in device mode.
```

Comment:
From a user perspective, this might be confusing. I think you can just say the attribute is ignored  on the host